libdnf-cli/output/transaction_table.hpp

```diff
@@ -294,6 +304,26 @@ bool print_transaction_table(Transaction & transaction) {
         scols_cell_set_color(ce, action_color(tspkg.get_action()));
 
         ts_summary.add(tspkg.get_action());
+        for (auto & replaced : tspkg.get_replaces()) {
+            struct libscols_line * ln_replaced = scols_table_new_line(tb, ln);
+            // TODO(jmracek) Translate it
+            std::string name("replacing ");
+            name.append(replaced.get_name());
+            scols_line_set_data(ln_replaced, COL_NAME, name.c_str());
+            scols_line_set_data(ln_replaced, COL_ARCH, replaced.get_arch().c_str());
+            scols_line_set_data(ln_replaced, COL_EVR, replaced.get_evr().c_str());
+            scols_line_set_data(ln_replaced, COL_REPO, replaced.get_repo_id().c_str());
+            uint64_t size = replaced.get_install_size();
+            scols_line_set_data(
+                ln_replaced, COL_SIZE, libdnf::cli::utils::units::format_size(static_cast<int64_t>(size)).c_str());
+
+            auto replaced_color = action_color(libdnf::transaction::TransactionItemAction::REPLACED);
+            scols_cell_set_color(scols_line_get_cell(ln_replaced, COL_EVR), replaced_color);
+            scols_cell_set_color(scols_line_get_cell(ln_replaced, COL_ARCH), replaced_color);
+            scols_cell_set_color(scols_line_get_cell(ln_replaced, COL_NAME), replaced_color);
+            scols_cell_set_color(scols_line_get_cell(ln_replaced, COL_REPO), replaced_color);
+            scols_cell_set_color(scols_line_get_cell(ln_replaced, COL_SIZE), replaced_color);
+        }
```

Comment:
I'm not insisting on this, but it would be nice to have a scenario or two in the CI stack that would test this. It would already make my life easier as I wouldn't have to test this manually, I could just run the tests...